hudi-cli/src/main/java/org/apache/hudi/cli/commands/BootstrapCommand.java

```diff
@@ -68,7 +68,7 @@ public String bootstrap(
           help = "Bootstrap Index Class") final String bootstrapIndexClass,
       @CliOption(key = {"selectorClass"}, unspecifiedDefaultValue = "org.apache.hudi.client.bootstrap.selector.MetadataOnlyBootstrapModeSelector",
           help = "Selector class for bootstrap") final String selectorClass,
-      @CliOption(key = {"keyGeneratorClass"}, unspecifiedDefaultValue = "org.apache.hudi.keygen.SimpleKeyGenerator",
+      @CliOption(key = {"keyGeneratorClass"}, unspecifiedDefaultValue = "org.apache.hudi.keygen.SparkSimpleKeyGenerator",
```

Comment:
This is what I meant in the first comment. we have to be backwards compatible with existing clients. It will be disruptive change for everyone to reconfigure all their jobs. Can we avoid renaming of keygenerator classes or changing their package names

Reply:
> This is what I meant in the first comment. we have to be backwards compatible with existing clients. It will be disruptive change for everyone to reconfigure all their jobs. Can we avoid renaming of keygenerator classes or changing their package names

got it. 
for speak, we leave the ket generator names untouched and rename it for other engines.